src/style.css

```diff
@@ -2,0 +1,39 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
```

Comment:
Don't fix container size (if there is no such a requirement). You've fixed the height and width of `.stars__star`. Ensure that this is a requirement of your design, otherwise consider using padding and allow the content to dictate the size to avoid overflow or accidental scroll bars.